packages/ui-kit/src/components/PieChart/PieChart.module.scss

```
@@ -0,0 +1,87 @@
+@use '../../themes/typography';
+@use '../../themes/common';
+
+.rootPieChart {
+    @extend .typographyTiny;
+    overflow: auto;
+    padding: 0;
+
+    canvas {
+        height: var(--propel-component-height);
+    }
+}
+
+.pieChartTotalValue{
+    text-align: center;
+    margin-top: 10px;
```

Please avoid using hardcoded values for `typography` and `spacing`. Please use design tokens instead, e.g.
```suggestion
    margin-top: var(--propel-space-sm);
```